project-security/verifying-signatures.md

```diff
@@ -660,11 +656,11 @@ the arguments to `gpg2`. (The signature file goes first.)
 
 There are several possibilities:
 - You don't have the [Qubes Master Signing
-  Key](#how-to-import-and-authenticate-the-qubes-master-signing-key).
-- [You have not set the Qubes Master Signing Key's trust level
+  Key (QMSK)](#how-to-import-and-authenticate-the-qubes-master-signing-key).
+- You have [not set the trust level of the QMSK
   correctly.](#how-to-import-and-authenticate-the-qubes-master-signing-key)
-- [In the case of a key that is not directly signed by the Qubes Master Signing
-  Key, you have not set that key's trust level
+- In the case of a key that is not directly signed by the QMSK,
+  you have [not set that key's trust level
```

Comment:
These were intentional, as a reader might find these FAQ entries without having been introduced to the acronym. Also, the people most likely to need these are the easily confused, and they may not pick up on the acronym. I'm open to be convinced otherwise, though.

Reply:
We did use the acronym throughout this page, but I didn't notice before that this would've been its first introduction into the FAQ.  I agree with your comment, I'm changing it back.